rails-commenting.rb

```diff
@@ -7,24 +7,29 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+# This populates when you create a controller, the controller handles all the RESTful actions of the application
 class BlogPostsController < ApplicationController
```

Comment:
What type of relationship is the < showing between BlogPostsController and Application Controller?